content/en/docs/installation.md

```diff
@@ -331,6 +332,75 @@ flux bootstrap gitlab \
   --path=clusters/my-cluster
 ```
 
+### Bitbucket Server
+
+The `bootstrap bitbucket-server` command creates a Bitbucket Server repository if one doesn't exist and
```

Comment:
We need to mention here that this will not work with BitBucket Cloud and link to some official Atlassian doc that explains the differences. What about Bitbucket Data Center will the bootstrap work with it?

Reply:
Bitbucket server and datacenter seems to run the same code:
https://confluence.atlassian.com/confeval/development-tools-evaluator-resources/bitbucket/bitbucket-bitbucket-server-vs-bitbucket-data-center

Reply:
Ok please rename the section to "Bitbucket Server and Data Center".